tests/api/ils/test_patrons.py

```diff
@@ -0,0 +1,83 @@
+import json
```

Comment:
great, thanks a lot for the test!!!

Please copy the header of the file from another test and check the imports.